### PR TITLE
[Merged by Bors] - Update operator-rs to 0.27.1

### PR DESCRIPTION
**Pull request by maltesander**
# Description

fixes https://github.com/stackabletech/opa-operator/issues/369

test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/opa-operator-it-custom/21/

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: rust/crd/src/lib.rs [hunk 1]
```diff
@@ -37,28 +49,40 @@ pub struct OpaSpec {
     pub version: Option<String>,
 }
 
-#[derive(Clone, Debug, Default, Deserialize, Eq, Merge, JsonSchema, PartialEq, Serialize)]
-#[serde(rename_all = "camelCase")]
+#[derive(Clone, Debug, Default, JsonSchema, PartialEq, Fragment)]
```
Comment (razvan) [inline rust/crd/src/lib.rs:52]:
```suggestion
#[allow(clippy::derive_partial_eq_without_eq)]
#[derive(Clone, Debug, Default, JsonSchema, PartialEq, Fragment)]
```

**Review by razvan — COMMENTED** [review]
Suggestion for fixing clippy warnings

**Review by razvan — APPROVED** [review]
great

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**